t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -245,10 +245,18 @@ sub javascript_console_has_no_warnings_or_errors {
             next if ($msg =~ qr,/thumb/, || $msg =~ qr,/.thumbs/,);
             # ignore error responses in 13-admin.t testing YAML errors
             next if ($msg =~ qr/api\/v1\/experimental\/job_templates_scheduling\/1003 - Failed to load resource/);
+            next if ($msg =~ qr/api\/v1\/experimental\/job_templates_scheduling\/1003 - Failed to load resource/);
+            # ignore when tests/:testid:/module_components_ajax is not found; the test modules might just not be available
+            # yet and running.js will just try to load them again
+            next if ($msg =~ qr/module_components_ajax - Failed to load resource/);
```

Comment:
does it make sense to generalize to `qr/_ajax…`?

Reply:
The exception is justified because the test modules might not be available directly from the start (the backend needs some time to generate them) and not because it is an AJAX error. I don't think it makes sense to ignore AJAX errors in general.